### PR TITLE
Import and use the CNCF Code of Conduct

### PR DESCRIPTION
**Pull request by cblecker**
This uses a generated import of the CNCF CoC

Fixes #416

### CODE REVIEW

**Comment by sftim** [comment]
Approach seems good to me.

**Comment by sftim** [comment]
I'd expect to see 2 or more commits, one of which actually does the import.

**Comment by sftim** [comment]
Want to put a Markdown file in source control but not have that render to its own web page? See https://github.com/kubernetes/website/pull/41793/files#diff-6837431359d119c18686bd0377481bf69921b81f4a221801d31e7fbe57aaa642 for an example of a way to do that:

```yaml
_build:
  list: never
  render: false
```

**Comment by cblecker** [comment]
@sftim This should be ready for review. Thanks!

**Comment by mrbobbytables** [comment]
> Want to put a Markdown file in source control but not have that render to its own web page? See https://github.com/kubernetes/website/pull/41793/files#diff-6837431359d119c18686bd0377481bf69921b81f4a221801d31e7fbe57aaa642 for an example of a way to do that:

Nice! I didn't know about that feature. 

Looks like a good method to pull this in and better than keeping a manual copy in repo 👍 
/approve

File: content/en/includes/_index.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+headless: true
```
Comment (sftim) [inline content/en/includes/_index.md:2]:
```suggestion
headless: true
_build:
  list: never
  render: false
```

**Review by sftim — CHANGES_REQUESTED** [review]
Almost.
https://deploy-preview-421--kubernetes-contributor.netlify.app/includes/ exists, and I don't think it should.

A similar story for https://deploy-preview-421--kubernetes-contributor.netlify.app/includes/cncf-code-of-conduct/

**Comment by sftim** [comment]
https://deploy-preview-421--kubernetes-contributor.netlify.app/includes/cncf-code-of-conduct/ still renders. Not the end of the world IMO.

/lgtm
/hold

This is an improvement and I'm OK to live with the snag that remains.

**Pull request merged**